pkg/daemon/update.go

```diff
@@ -541,6 +525,15 @@ func (dn *Daemon) update(oldConfig, newConfig *mcfgv1.MachineConfig) (retErr err
 	oldConfigName := oldConfig.GetName()
 	newConfigName := newConfig.GetName()
 
+	oldIgnConfig, err := ctrlcommon.ParseAndConvertConfig(oldConfig.Spec.Config.Raw)
+	if err != nil {
+		return fmt.Errorf("parsing old Ignition config failed with error: %v", err)
```

Comment:
Instead of `%v`, use `%w`. This will allow the error from `ctrlcommon.ParseAndConvertConfig()` to be cleanly wrapped inside the error this returns. This will be useful at a later date when we revamp how we handle errors.

Reply:
Looks like I still have to use `%v` with `t.Errorf` though?